### ISSUE TITLE
Who brings what?

### ISSUE THREAD

**Issue by nanoparsec** [rubytown/32c3#2]
If you are planning to bring you coffee maker, cofee beans, holiday lights and other things to make our assembly nice and cosy, please leave a comment here. <3

**Comment by moonglum** [comment]
YAY Coffee :coffee: 

Will definitely bring coffee beans. I think our coffee creation process last year (Water Boiler + Chemex + Hand Grinder) was good.

**Comment by bitboxer** [comment]
We need:
- [x] Powerplugs
- [x] Network switch
- [x] Long network cables (>10m)

**Comment by cashewkernchen** [comment]
I will definitely bring:
- some network cables
- Coffeeee (beans)
- Coffee filters
- and a mug

**Comment by thilo** [comment]
I will bring the posters from last year :)

**Comment by cashewkernchen** [comment]
We need: 
- a DECT 

I registered a DECT number at [Eventphone](https://eventphone.de/guru2/phonebook?event=32C3&s=ruby_town), so we could use 7829 / RUBY for a central DECT at our assembly

**Comment by bitboxer** [comment]
I will bring some power plugs, my dect and network cables. But a switch would be awesome to have

![images duckduckgo](https://cloud.githubusercontent.com/assets/56195/11920019/2c9762f6-a763-11e5-93f9-6342a45a1688.jpg)

**Comment by moonglum** [comment]
We have no switch so far, does @cashewkernchen have one she could bring? :smile_cat:

**Comment by schultyy** [comment]
I bring a power plug. Maybe some network cables, too.

**Comment by cashewkernchen** [comment]
Yes, I can bring a 16 port Gigabit Switch if Rubytown takes good care of it.

We still need a water boiler iirc. Anyone?

**Comment by moonglum** [comment]
:dancers:

**Comment by moonglum** [comment]
We have @railsbros-dirk's waterboiler :smile:

**Comment by schultyy** [comment]
Need to check but maybe I can also bring a small 8 port switch

**Comment by code-later** [comment]
I'll bring a 16-port switch and a 10m cable.

**Comment by code-later** [comment]
And my Chemex as well. And coffee grinder😉

**Comment by nanoparsec** [comment]
I will bring some holiday lights for atmosphere :) cookies. tea and a lot of Rails Girls Berlin cups, posters, stickers and buttons.

**Comment by bitboxer** [comment]
I will bring 20m Cat6 and 7m cat6 and a few 2m ones. and 10m power cable + a 6-socket outlet thingy.

**Comment by klaustopher** [comment]
I will be travelling to Hamburg tomorrow ... Is anything else needed?

**Comment by cashewkernchen** [comment]
Maybe some tools like scissors, adhesive tapes and some cords. At least thats what we lacked last year when building up everything.
